app/views/coronavirus_local_restrictions/results.html.erb

```diff
@@ -17,12 +17,14 @@
     %>
     <% if !@search.location_lookup.data.first.england? %>
       <%= render partial: "coronavirus_local_restrictions/devolved_nation", locals: locals %>
+    <% elsif @search.local_restriction&.tier_one? %>
+      <%= render partial: "coronavirus_local_restrictions/tier_one_restrictions", locals: locals %>
     <% elsif @search.local_restriction&.tier_two? %>
       <%= render partial: "coronavirus_local_restrictions/tier_two_restrictions", locals: locals %>
     <% elsif @search.local_restriction&.tier_three? %>
       <%= render partial: "coronavirus_local_restrictions/tier_three_restrictions", locals: locals %>
     <% else %>
-      <%= render partial: "coronavirus_local_restrictions/tier_one_restrictions", locals: locals %>
+      <%= render partial: "coronavirus_local_restrictions/no_tier_information" %>
```

Comment:
There's already a template for no tier information. Can that be used instead? Perhaps some of the common elements could be moved into a partial?

Reply:
I made a separate method no_restriction_information_england_postcode? and then used that in the controller to render the view, rather than rendering it as a partial from the results view.